src/content/docs/alerts-applied-intelligence/applied-intelligence/incident-intelligence/change-applied-intelligence-correlation-logic-decisions.mdx

```diff
@@ -957,51 +964,62 @@ Here are technical details on the similarity algorithms we use:
   </Collapser>
 </CollapserGroup>
 
-## Regex operators [#regex]
+### Regex operators [#regex]
 
-When [building a decision,](#customize) available operators include:
+When [building a decision](#customize), available operators include:
 
 * `contains (regex)`: used in [Step 1: Filter your data](#customize).
 * `regular expression match`: used in [Step 2: Contextual correlation](#customize).
 
 The decision builder follows the standards outlined in [these documents for regular expressions](https://docs.oracle.com/javase/8/docs/api/java/util/regex/Pattern.html).
 
-### Regex in Step 1 [#regex-step-1]
-
-In order for your regex to test as true, the entire attribute value (the data you're evaluating) must be matched by the regular expression provided. Captured groups can be used but are not explicitly evaluated.
-
-For instance, if the attribute value is `foobarbaz`, these examples would meet the criteria and test as true:
-
-* `foo.*`
-* `^.*baz`
-* `\w+`
+<CollapserGroup>
+  <Collapser
+    id="regex-step-1"
+    title="Regex in Step 1"
+  >
+    In order for your regex to test as true, the entire attribute value (the data you're evaluating) must be matched by the regular expression provided. Captured groups can be used but are not explicitly evaluated.
 
-### Regex in Step 2 [#regex-step-2]
+    For instance, if the attribute value is `foobarbaz`, these examples would meet the criteria and test as true:
 
-In order for your regex to test as true, the entire attribute values for incident 1 and incident 2 must be included in the match. Also, each captured group (expressions in `( )` parentheses) must exist in both values (incident 1 and incident 2 attributes), and have the same value:
+    * `foo.*`
+    * `^.*baz`
+    * `\w+`
+  </Collapser>
 
-* The number of captured groups must be equal for both incident attributes.
-* Each group must be equal to the corresponding group between attribute values: the value of the first captured group in the incident 1 attribute value is equal to the value of the first captured group in the incident 2 attribute.
+  <Collapser
+    id="regex-step-2"
+    title="Regex in Step 2"
+  >
+    In order for your regex to test as true, the entire attribute values for incident 1 and incident 2 must be included in the match. Also, each captured group (expressions in `( )` parentheses) must exist in both values (incident 1 and incident 2 attributes), and have the same value:
 
-For instance, if attribute value 1 is `abc-123-xyz` and attribute value 2 is `abc-777-xyz`, then `(\w+)-(?:\w+)-(\w+)` would meet the criteria:
```

Comment:
i haven't checked it in build yet, but you might want to review how these look in the build and see if the indents look okay.